core/txpool/legacypool/legacypool.go

```diff
@@ -568,9 +572,19 @@ func (pool *LegacyPool) Pending(filter txpool.PendingFilter) map[common.Address]
 				}
 			}
 		}
+		if filter.MaxDATxSize != nil && !pool.locals.contains(addr) {
```

Comment:
nit, more similar to other checks
```suggestion
		if maxDASizeScaled != nil && !pool.locals.contains(addr) {
```

Reply:
now that I changed it to use non-scaled everywhere, I dropped the maxDASizeScaled var entirely, so leaving this as is.